### PR TITLE
Added FAQ

### PR DESCRIPTION
**Pull request by HuiiBuh**

### CODE REVIEW

**Comment by HuiiBuh** [comment]
I only documented the problems I had getting started with mkdocstrings and wrote them in the FAQ.md

**Comment by pawamoy** [comment]
Hey, this is great, thank you so much!

Could you directly put the contents in `docs/faq.md`? I don't want/need it to be in the root of the repo as `FAQ.md`. Just personal taste.

Also, in the `nav`, could you rename it from `FAQ` to `Frequently Asked Questions`? Thanks!

I'll merge as it is right now and tweak a bit the text later :slightly_smiling_face:

**Comment by HuiiBuh** [comment]
I have made the changes you proposed

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# FAQ
```
Comment (StephenBrown2) [inline FAQ.md]: Also delete this file, as it's a duplicate of `docs/faq.md`

**Comment by HuiiBuh** [comment]
Oh I thought that I did delete that. Should be fixed

**Comment by pawamoy** [comment]
Merging, thanks again!

**Pull request merged**